apps/userdashboard/views.py

```diff
@@ -55,9 +59,21 @@ def actions(self):
     def projects_carousel(self):
         sorted_active_projects, sorted_future_projects, sorted_past_projects =\
             self.request.user.get_projects_follow_list()
-        return (list(sorted_active_projects) +
-                list(sorted_future_projects) +
-                list(sorted_past_projects))[:8]
+        display_projects = (list(sorted_active_projects) +
+                            list(sorted_future_projects) +
+                            list(sorted_past_projects))[:8]
+        projects = []
+        for project in display_projects:
+            project.latest_comments_num = self._latest_comments_num(
+                project)
+            projects.append(project)
+
+        return projects
+
+    def _latest_comments_num(self, project):
+        all_comments_project = helpers.get_all_comments_project(project)
+        return all_comments_project.filter(created__gte=datetime.now() -
+                                           timedelta(days=7)).count()
```

Comment:
do we want to show the most comments or the most recent? it's currently serving the project with the most comments first, not the project with most recent comments, don't know if we care?!